src/humancellatlas/data/metadata/api.py

```diff
@@ -620,7 +626,12 @@ class File(LinkedEntity):
     def __init__(self, json: JSON, manifest: Mapping[str, ManifestEntry]):
         super().__init__(json)
         content = json.get('content', json)
+        # '/' was once forbidden in file paths and was encoded with '!'. Now
+        # '/' is allowed and we force it in the metadata so that backwards
+        # compatibility is simplified downstream.
+        # core = copy.deepcopy(content['file_core'])
         core = content['file_core']
+        core['file_name'] = core['file_name'].replace('!', '/')
```

Comment:
I get a pycharm warning here. See comment on cast above.

Reply:
Strangely I did not get a warning.

Reply:
This should be fixed by changing the JSON definition.